src/common-elements/tabs.ts

```diff
@@ -1,5 +1,6 @@
 import { Tabs as ReactTabs } from 'react-tabs';
 import styled from '../styled-components';
+import { darken, lighten } from 'polished';
```

Comment:
From Travis-CI:

ERROR: /home/travis/build/Rebilly/ReDoc/src/common-elements/tabs.ts[3, 1]: Import sources within a group must be alphabetized.